ckanext/validation/plugin/__init__.py

```diff
@@ -344,3 +349,37 @@ def _get_underlying_file(wrapper):
         return wrapper.stream
     return wrapper.file
 
+
+def _should_remove_unsupported_resource_validation_reports(res_dict):
+    if not t.h.asbool(t.config.get('ckanext.validation.clean_validation_reports', False)):
+        return False
+    return ((not res_dict.get('format', u'').lower() in settings.SUPPORTED_FORMATS
+                or res_dict.get('url_changed', False))
```

Comment:
I don't see how `url_changed` is relevant in this context.

Reply:
Removed `url_changed` check